python/Ice/greeter/README.md

```diff
@@ -47,8 +57,18 @@ In a separate terminal, navigate to the `client` directory to run the client pro
 
     ```shell
     python -m venv venv
```

Comment:
On a regular linux install there don't be a `python` command. I'd probably duplicate this command to use python on windows and python3 and macOS and linux